Update Build Scan wording (#33752)
gradle/gradle: platforms/documentation/docs/src/docs/userguide/running-builds/introduction/build_scans.adoc

// limitations under the License.

[[build_scans]]
= Build Scan

++++
<div class="badge-wrapper">
    <a class="badge" href="https://dpeuniversity.gradle.com/app/courses/b5069222-cfd0-4393-b645-7a2c713853d5/" target="_blank">
        <span class="badge-type button--blue">LEARN</span>

>         <span class="badge-text">How to Use a Build Scan&nbsp;&nbsp;&nbsp;&gt;</span>

Reviewer: ```suggestion
        <span class="badge-text">How to Use Build Scan&nbsp;&nbsp;&nbsp;&gt;</span>
```

    </a>
</div>
++++

A Build Scan is a *representation of metadata captured* as you run your build.

image::gradle-basic-1.png[]

== About Build Scan

Gradle <<#sec:captured-information,captures your build metadata>> and sends it to the link:https://scans.gradle.com/[Build Scan Service].
The service then transforms the metadata into information you can analyze and share with others.

image::build-scan-1.png[]

> The information that Build Scans collect can be an invaluable resource when troubleshooting, collaborating on, or optimizing the performance of your builds.

Reviewer: ```suggestion
The information that Build Scan collects can be an invaluable resource when troubleshooting, collaborating on, or optimizing the performance of your builds.
```

For example, with a Build Scan, it’s no longer necessary to copy and paste error messages or include all the details about your environment each time you want to ask a question on Stack Overflow, Slack, or the Gradle Forum.
Instead, copy the link to your latest Build Scan.

image::build-scan-2.png[]

> == Enable a Build Scan

Reviewer: ```suggestion
== Enable Build Scan
```

To enable a Build Scan on a Gradle command, add `--scan` to the command line option:

[source,text]
----
 ./gradlew build --scan
----

> You may be prompted to agree to the terms to use a Build Scan.

Reviewer: ```suggestion
You may be prompted to agree to the terms to use Build Scan.
```
